app/src/renderer/views/SignIn.tsx

```diff
@@ -0,0 +1,236 @@
+import { Button, Input, Form } from "antd";
+import { EyeInvisibleOutlined, EyeTwoTone } from "@ant-design/icons";
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router";
+
+import type { ProxyRequest, ProxyJSONResponse } from "../../types";
+import { useAppDispatch } from "../hooks";
+import type { SessionState } from "../features/session/sessionSlice";
+import { set, clear } from "../features/session/sessionSlice";
+import logoImage from "../../../resources/images/logo.png";
+
+type FormValues = {
+  username: string;
+  passphrase: string;
+  oneTimeCode: string;
+};
+
+function SignInView() {
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+
+  const [form] = Form.useForm();
+  const [version, setVersion] = useState<string>("");
+  const [authError, setAuthError] = useState<boolean>(false);
+  const [isSubmitting, setIsSubmitting] = useState<boolean>(false);
```

Comment:
Cross-referencing <https://github.com/freedomofpress/securedrop-client/pull/2543#issuecomment-3074195850>, where I've since found Redux's suggested action/reducer semantics for an authentication state machine:

https://redux.js.org/faq/miscellaneous#how-can-i-implement-authentication-in-redux